parachain-tracer/src/tracker.rs

```diff
@@ -299,21 +349,18 @@ impl SubxtTracker {
 		bitfields: Vec<AvailabilityBitfield>,
 		storage: &TrackerStorage,
 	) -> color_eyre::Result<()> {
-		if self.current_candidate.is_backed() {
-			// We only process availability if our parachain is assigned to an availability core.
-			if let Some(core) = self.current_candidate.assigned_core {
-				self.current_candidate.current_availability_bits = extract_availability_bits_count(bitfields, core);
-				self.current_candidate.max_availability_bits =
-					self.validators_indices(block_hash, storage).await?.len() as u32;
-
-				if self.current_candidate.is_data_available() {
-					self.current_candidate.set_included();
-					self.relay_forks.last_mut().expect("must have relay fork").included_candidate =
-						self.current_candidate.candidate_hash;
-					self.previous_included_at = self.last_included_at;
-					self.last_included_at = self.current_relay_block.map(|v| v.into());
-				} else if !self.is_just_backed() {
-					self.current_candidate.set_pending();
+		let core_ids: Vec<u32> = self.cores.keys().cloned().collect();
+		for core in core_ids {
+			if self.is_current_candidate_backed(core) {
+				let is_just_backed = self.is_just_backed();
+				let max_bits = self.validators_indices(block_hash, storage).await?.len() as u32;
+				let candidate = self.current_candidate_mut(core).expect("Checked above");
+
+				candidate.current_availability_bits = extract_availability_bits_count(&bitfields, core);
```

Comment:
If the candidate was backed at current block, the bits should not be set at all. I'd assume we want to do this only if `!is_just_backed`, below.